kafka-java-liberty-sample/src/main/java/com/eventstreams/samples/servlet/KafkaServlet.java

```diff
@@ -332,12 +384,12 @@ public void run() {
             while (!closing) {
                 try {
                     currentConsumedMessage = "consumer is waiting for messages to be consumed ...";
-                    // Poll on the Kafka consumer every second.
-                    Iterator<ConsumerRecord<String, String>> it = kafkaConsumer.poll(1000).iterator();
+                    Duration duration = Duration.of(1, ChronoUnit.SECONDS);
```

Comment:
`Duration.ofSeconds()`?

Reply:
The previous code was using a deprecated method.

Reply:
I understand `poll()` with a `long` is now deprecated but I suggested using `Duration.ofSecond(1L)` instead of `Duration.of(1, ChronoUnit.SECONDS)`